tests/tasks/test_http.py

```diff
@@ -126,3 +127,30 @@ def test_http_get_define():
     ):
         http = Http(name, url)
         assert http.task_params == expect_task_params
+
+
+def test_http_params_conversion():
+    # Create a sample http_params dictionary
+    http_params_dict = {
+        "prop1": "value1",
+        "prop2": "value2",
+        "prop3": "value3"
```

Comment:
it is better to add more value type for our test
```suggestion
        "prop1": "value1",
        "prop2": 135,
        "prop3": "value3"
```